contrib/win32/win32compat/ssh-agent/agent.h

```diff
@@ -38,6 +38,8 @@ struct agent_connection {
 	} client_type;
 };
 
+char* sshagent_con_username;
```

Comment:
This will create multiple instances of `sshagent_con_username` in every .c that includes agent.h. Perhaps move to agent.c?

Reply:
fixed